edi_oca/models/edi_backend.py

```diff
@@ -582,6 +610,33 @@ def _exchange_receive_check(self, exchange_record):
             "input_receive_error",
         ]
 
+    def exchange_send_on_max_retries_reached(self, exchange_records):
```

Comment:
good catch, I didn't foresee that `exchange_send` could be run as a job!
but did we ever faced an error in that job? doesn't it just create other jobs?

Reply:
Hi nils, did you mention [this one](https://github.com/OCA/edi-framework/commit/6f8d5e5b6a9db6c53f5d4fce549ac6b5a613e13a)?

Reply:
- actually since [this commit](https://github.com/OCA/edi-framework/commit/96827f0f1fe48520b70a4ded1f572c3786465459), these methods are not run directly as jobs anymore
- they are called by `action_exchange_send`, `action_exchange_send`, `action_exchange_receive`
- so you can remove these 3 hooks
- also, as an additional task on same ticket, can you please prepare an aditional PR on edi_oca to remove them from job_function + tests as it was [noted here](https://github.com/camptocamp/edi-framework/blob/452b75b42ef3948fc2efaf02bc7ed296cef53408/edi_oca/data/job_function.xml#L27)

Reply:
wow, missing point, thank nils

I didn't read the code carefully

In tests, Sima used `with_delay` with `backend`, so I assumed that jobs were applied on `backend`. Thanks for your discovering